_src/static/sass/utilities/_helpers.scss

```diff
@@ -0,0 +1,139 @@
+// ------------------------------
+// edX Pattern Library: Helpers
+
+// About: Sass partial for defining extends and re-usable elements at the Sass compile level.
+
+// #UTILITIES
+// #RESETS
+
+// ----------------------------
+// #UTILITIES
+// ----------------------------
+// visual link
+%fake-link {
```

Comment:
I'd prefer to remove it - it's only a single style, after all - and define `cursor: pointer` on an as-needed basis. You cool with that?